src/Public/OpenApi.ps1

```diff
@@ -252,7 +276,9 @@ function Add-PodeOAServerEndpoint {
         [string]
         $Url,
         [string]
-        $Description
+        $Description,
+        [System.Collections.Specialized.OrderedDictionary]
+        $Variables
```

Comment:
Could this not just be a simple `[hashtable]`?

Reply:
if is not ordered the the generated openapi document is messy